### PR TITLE
doc: add Upgrading Instruction for v1.8 to v1.9

### PR DESCRIPTION
**Pull request by XiShanYongYe-Chang**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

doc: add Upgrading Instruction for v1.8 to v1.9

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

The upgrade process has been verified.

### CODE REVIEW

**Comment by RainbowMango** [comment]
This should also added to the v1.9 release(not only the current release), right?

**Comment by XiShanYongYe-Chang** [comment]
> This should also added to the v1.9 release(not only the current release), right?

It has been updated.

File: docs/administrator/upgrading/v1.8-v1.9.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+* Introduced `ActivationPreference` to the `PropagationPolicy` and `ClusterPropagationPolicy` to indicate how the referencing resource template will be propagated, in case of policy changes.
+* Introduced the `Remedy` CRD in the `remedy.karmada.io` group.
+* Introduced `RemedyActions` to the `Cluster` API to represent the remedy actions.
+* Introduced `TrafficBlockClusters` and `ServiceLocations` to the `MultiClusterIngress` API.
+* Added additional printer columns KIND for `Work` CRD.
+
+### karmada-search
+
+* Support accept content type as=Table in the proxy global resource function.
```
Comment (windsonsea) [inline docs/administrator/upgrading/v1.8-v1.9.md:19]:
```suggestion
* The Proxy global resource feature supports content type `=Table`.
```

File: versioned_docs/version-v1.9/administrator/upgrading/v1.8-v1.9.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+* Introduced `ActivationPreference` to the `PropagationPolicy` and `ClusterPropagationPolicy` to indicate how the referencing resource template will be propagated, in case of policy changes.
+* Introduced the `Remedy` CRD in the `remedy.karmada.io` group.
+* Introduced `RemedyActions` to the `Cluster` API to represent the remedy actions.
+* Introduced `TrafficBlockClusters` and `ServiceLocations` to the `MultiClusterIngress` API.
+* Added additional printer columns KIND for `Work` CRD.
+
+### karmada-search
+
+* Support accept content type as=Table in the proxy global resource function.
```
Comment (windsonsea) [inline versioned_docs/version-v1.9/administrator/upgrading/v1.8-v1.9.md:19]:
```suggestion
* The Proxy global resource feature supports content type `=Table`.
```

File: docs/administrator/upgrading/v1.8-v1.9.md (hunk 1 above)
Comment (XiShanYongYe-Chang) [reply docs/administrator/upgrading/v1.8-v1.9.md]: Thanks a lot~
updated.

**Comment by windsonsea** [comment]
Thanks.
/lgtm

**Review by RainbowMango — APPROVED** [review]
/approve

**Pull request merged**